src/vs/workbench/contrib/debug/browser/welcomeView.ts

```diff
@@ -62,7 +62,11 @@ export class WelcomeView extends ViewPane {
 		this.debugStartLanguageContext.set(lastSetLanguage);
 
 		const setContextKey = () => {
-			const editorControl = this.editorService.activeTextEditorControl;
+			let editorControl = this.editorService.activeTextEditorControl;
+			if (isDiffEditor(editorControl)) {
+				editorControl = editorControl.getOriginalEditor();
```

Comment:
Shouldn't this look at the modified editor? I think I want to debug the one on the right.

Reply:
That's a good point, I missed the possibility that the user might edit the language for the file, updated.